openslides/users/config_variables.py

```diff
@@ -90,3 +92,43 @@ def get_config_variables():
         weight=570,
         group='Participants',
         subgroup='PDF')
+
+    # Email
+
+    yield ConfigVariable(
+        name='users_email_sender',
+        default_value='noreply@yourdomain.com',
+        input_type='string',
+        label='Email sender',
+        weight=600,
+        group='Participants',
+        subgroup='Email')
+
+    yield ConfigVariable(
+        name='users_email_subject',
+        default_value='Your login for {event_name}',
+        input_type='string',
+        label='Email subject',
+        help_text='You can use {event_name} as a placeholder.',
+        weight=605,
+        group='Participants',
+        subgroup='Email')
+
+    yield ConfigVariable(
+        name='users_email_body',
+        default_value=dedent('''\
```

Comment:
@normanjaeckel This `dedent` does not work. The first line is indended, but the rest not:
```
Dear Administrator,

        this is your OpenSlides login for the event "OpenSlides":

            http://example.com:8000
            username: admin
            password: admin

        This email was generated automatically.
```
Can you verify this?

Reply:
Code looks good. Maybe it is hacked somewhere else?

Reply:
Maybe you have to indent all lines again by 4 spaces.

Reply:
Yep!